### PR TITLE
Automl links fix

### PR DESCRIPTION
**Pull request by krishr2d2**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/sdk_automl_video_classification_batch.ipynb [hunk 1]
```diff
@@ -3,12 +3,13 @@
     {
```
Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Do not update copyright year</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Move Dataset below objectiv</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Use notebook template.</p><p>Combine installs into single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Use template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>use template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Vertex SDK -&gt; Vertex AI SDK</p><p> </p><p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Replace GAPIC version with SDK version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Vertex model -&gt; Vertex AI Model resource</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: <p>Model resource -&gt; Vertex AI Model resource</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: updated...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: updated..

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: moved...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: updated...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: done...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: updated...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: done..

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: done...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: done...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: Replaced...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: updated...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: added...

Comment (krishr2d2) [reply notebooks/official/automl/sdk_automl_video_classification_batch.ipynb:3]: updated...

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**